packages/react-components/src/VoteValue.tsx

```diff
@@ -55,7 +57,7 @@ function getValues (selectedId: string | null | undefined, noDefault: boolean |
     })
     .map(({ amount }) => amount);
 
-  const maxValue = allBalances.votingBalance;
+  const maxValue = isReferenda && api.query.convictionVoting ? allBalances.votingBalance.add(allBalances ? allBalances.reservedBalance : new BN(0)) : allBalances?.votingBalance;
```

Comment:
```suggestion
  const maxValue = isReferenda && api.query.convictionVoting ? allBalances.votingBalance.add(allBalances.reservedBalance) : allBalances?.votingBalance;
```
Why are we checking for `allBalances` inside `allBalances.votingBalance.add`?

Reply:
Since the useCall returns `<T> | undefined`

Reply:
This one actually doesnt need it because allBalances is not going to be undefined since its checked in the component. Fixed this one.